lowes.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+import json 
+
+states = ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DC", "DE", "FL", "GA", 
```

Comment:
This looks like an extra file. This should probably be in `locations/spiders/`?

Reply:
Deleted the extra file.